src/Parsers/ParserDropQuery.cpp

```diff
@@ -37,15 +41,23 @@ bool parseDropQuery(IParser::Pos & pos, ASTPtr & node, Expected & expected, cons
     ASTPtr database;
     ASTPtr database_and_tables;
     String cluster_str;
+    ASTPtr like;
     bool if_exists = false;
     bool if_empty = false;
     bool has_all_tables = false;
```

Comment:
Maybe also change this as we are doing `query->has_tables = has_all_tables;` on line 157?
```suggestion
    bool has_tables = false;
```